python/ingestion/constants.py

```diff
@@ -52,6 +52,66 @@
     "Wyoming"
 ]
 
+STATE_NAMES_TO_FIPS = {
```

Comment:
Rather than hard-coding a mapping here, IMO it would be better to do the state name -> FIPS conversion in an SQL aggregation routine for now, similar to what we do for CDC restricted here (join with a public bq table which has state names & fips):

https://github.com/SatcherInstitute/health-equity-tracker/blob/master/config/data_sources/AGG_cdc_restricted.sql#L22